### ISSUE TITLE
Unexpected indentation in multi-arity `defn`

### ISSUE THREAD

**Issue by alexandercampbell** [oakmac/standard-clojure-style-js#99]
This file:

```clojure
(ns example)

(defn rgb->css
  ([r g b] (str "rgb(" r "," g "," b ")"))
  ([r g b a] (str "rgba(" r "," g "," b "," a ")")))
```

lIs reformatted by standard-clj v0.4.1 like so:

```clojure
(ns example)

(defn rgb->css
  ([r g b] (str "rgb(" r "," g "," b ")"))
([r g b a] (str "rgba(" r "," g "," b "," a ")")))
```

I would expect the last line to be indented.

I took this example from the middle of a larger file, so I know that the odd indentation is not related to this being the last form in a file.

Notably, this altered version is correctly formatted as I expect:
```clojure
(ns example)

(defn rgb->css
  ([r g b] (str))
  ([r g b a] (str)))
```

I suspect the bug is connected to parentheses appearing inside string literals.

I observed this behavior on v0.4.1 of the formatter.

**Comment by oakmac** [comment]
Great catch! Thank you for the helpful bug report 🤓 Fixed with [PR-100](https://github.com/oakmac/standard-clojure-style-js/pull/100)

**Comment by oakmac** [comment]
Published v0.5.0 with this fixed
